client/src/components/ContactUs/index.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Box, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { Facebook, LinkedIn, Instagram, Twitter } from '@material-ui/icons';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Since the styles are the longest part of this component file, you could reorganize them into their own file and import them here.  It's not super important but it'd be helpful for the next person who had to change this component.